### DISCUSSION TITLE
Compatibility with Bond Pro?

### DISCUSSION THREAD

**Discussion by imatarasso** [aarons22/homebridge-bond#147] [General]
Hello - I previously used this plug-in successfully with my Bond. I've just migrated to the new Bond-Pro and I am wondering if you are looking for any testing with the Bond Pro?

**Comment by aarons22** [comment]
Definitely! I just got a bond pro but haven't had a chance to get it setup. If you have any issues, please let me know!

**Reply by PXIII** [reply]
Hi!
Does this plugin work with Bond Pro, please? Wouldn’t like to order a $379 Bond device if it doesn’t …
Cheers!

**Reply by aarons22** [reply]
Hey there! So, I never got around to testing the Bond Pro with this plugin. I'm hoping someone else has by this point....

I can't commit to doing that right away either, but I'm hoping I can in the near future

**Reply by imatarasso** [reply]
I own a Bond Pro and use it with this HomeBridge plug-in over the last few months. It has worked just fine. 

If the Bond has a hiccup (for whatever reason), the plug-in doesn't work either (of course).  Otherwise, I've been very satisfied and appreciate the efforts to produce this plug-in.
